WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginBrowserActivity.java

```diff
@@ -82,9 +90,10 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        ((WordPress)getApplication()).component().inject(this);
```

Comment:
I believe, this was the missing thing when I worked on it. I guess we'd need that for the `ViewModelProvider.Factory` to be injected.

Reply:
Activities and Fragments are created by the Android Framework, so we need to add them into the Dagger dependency graph manually, in order to enable field injection.